### PR TITLE
VA letters scaffolding

### PR DESCRIPTION
**Pull request by annekainicUSDS**
Closes https://github.com/department-of-veterans-affairs/vets.gov-team/issues/2793

PR for the VA letters scaffolding. Added basic files, like entry, routes, and containers. Added path to ignoreList and entry file list.

Need to have a better sense of what the data store looks like before adding more detail here.

### CODE REVIEW

File: src/js/va-letters/va-letters-entry.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  basename: '/va-letters'
+});
+
+function init() {
+  ReactDOM.render((
+    <Provider store={commonStore}>
+      <Router history={history}>
+        <Route path="/" component={VALettersApp}>
+          <IndexRedirect to="/confirm-address"/>
+          {routes}
```
Comment (raquelromano-gov) [inline src/js/va-letters/va-letters-entry.jsx]: Not sure why but it looks like the other components (`ConfirmAddress` and `DownloadLetters`) aren't getting rendered the `confirm-address` and `download-letters` endpoints  are loaded. I'm glad to look at it more this afternoon.

File: src/js/va-letters/va-letters-entry.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+const history = useRouterHistory(createHistory)({
+  basename: '/va-letters'
+});
+
+function init() {
+  ReactDOM.render((
+    <Provider store={commonStore}>
+      <Router history={history}>
+        <Route path="/" component={VALettersApp}>
+          <IndexRedirect to="/confirm-address"/>
```
Comment (annekainicUSDS) [inline src/js/va-letters/va-letters-entry.jsx]: Looks at the mockups, I don't think there is an intro page. I think you just go from your profile page to the first page where you confirm your address. I could be missing something though!

File: src/js/va-letters/va-letters-entry.jsx (hunk 1 above)
Comment (annekainicUSDS) [inline src/js/va-letters/va-letters-entry.jsx]: Hm that's odd, it was working for me locally. Maybe try again after running `npm run build` and restarting the `watch` task?

File: src/js/va-letters/va-letters-entry.jsx (hunk 2 above)
Comment (jcmeloni-usds) [inline src/js/va-letters/va-letters-entry.jsx]: I wouldn't stress too hard about this right now -- I haven't even approved a mockup/flow version yet.  Just fyi.

File: src/js/va-letters/va-letters-entry.jsx (hunk 1 above)
Comment (raquelromano-gov) [inline src/js/va-letters/va-letters-entry.jsx]: Oh, I was missing your latest commit. Looks good!

**Review by raquelromano-gov — APPROVED** [review]

**Comment by annekainicUSDS** [comment]
I don't have the right permissions to resolve the Code Climate issue. Going to merge for now without it being resolved, even though merging a PR with a failed check gives me existential angst.

**Pull request merged**